core/src/test/scala/unit/kafka/server/ProduceRequestTest.scala

```diff
@@ -84,14 +86,41 @@ class ProduceRequestTest extends BaseRequestTest {
       new SimpleRecord(System.currentTimeMillis(), "key2".getBytes, "value2".getBytes)), 1)
   }
 
-  @ParameterizedTest
+  private def getPartitionToLeader(
+    admin: Admin,
+    topic: String
+  ): Map[Int, Int] = {
+    var topicDescription: TopicDescription = null
+    TestUtils.waitUntilTrue(() => {
+      val topicMap = admin.
+        describeTopics(java.util.Arrays.asList(topic)).
+          allTopicNames().get(10, TimeUnit.MINUTES)
+      topicDescription = topicMap.get(topic)
+      topicDescription != null
+    }, "Timed out waiting to describe topic " + topic)
+    topicDescription.partitions().asScala.map(p => {
+      p.partition() -> p.leader().id()
+    }).toMap
+  }
+
+  @ParameterizedTest(name = "quorum=kraft")
```

Comment:
This is just to indicate in the name the test is running via kraft? Just double checking since the quorum is not a parameter in the method.

Reply:
the test name is what TestInfo.scala looks at to determine if the test is kraft. Changing the test name was just an easy way of getting it to be kraft

Reply:
Ok -- just wasn't sure if we also needed the quorum param.

Reply:
in this case, no, since we changed the test name directly. sorry I guess that was a bit obscure :)

soon, we will have everything as kraft and we can get rid of this.